### PR TITLE
Work Section

### PR DESCRIPTION
**Pull request by basnetlaxmi**
- I added Projects section .
- I added About Me section.
- I added some screenshots of my project in README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css" />
     <title>Document</title>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Firstly, Let us use the descriptive title for our project, `for example your full name not Document`

File: index.html [hunk 2]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
 
 
 
         </section>
+    
+        
+    
+        
+        <section  class= "work-container">
+            <h2 class="title"> My Recent Works</h2>
```
Comment (Emmyn5600) [inline index.html]: - [ ] This should be `h1` not `h2`

File: index.html [hunk 3]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
+                     industry's standard dummy text ever since the 1500s, when an unknown printer took a 
+                     standard dummy text.
+                </p>
+
+                <div class="languages">
+                    <a href="#" class="language">css</a>
+                    <a href="#" class="language"> html</a>
+                    <a href="#" class="language">bootstrap</a>
+                    <a href="#" class="language"> ruby</a>
+                </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 4]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 5]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 6]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 7]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
+                        </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 8]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 9]
```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
...
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @basnetlaxmi 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 

- Linters are passing successfully:heavy_check_mark:
- You have the descriptive PR:heavy_check_mark:
- You have the descriptive commits:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:
- You have the descriptive Readme file:heavy_check_mark:
- You followed the git-flow:heavy_check_mark:
   
Changes required :writing_hand: 

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Murodjon000 — APPROVED** [review]
**Hi Laxmi Basnet 👋,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**